class Error(Exception): pass
class InsertError(Error): pass
